modules/appeals_api/config/shared_schemas/address.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Let's bump the schemas to the latest version.

Reply:
draft-07 byeeeee